<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>bookOp</class>
 <widget class="QDialog" name="bookOp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>663</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>610</y>
     <width>392</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="insertBTN">
      <property name="text">
       <string>插入</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="changeBTN">
      <property name="text">
       <string>修改</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="delBTN">
      <property name="text">
       <string>删除</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="clearBtn">
      <property name="text">
       <string>清空</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>555</y>
     <width>45</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>出版社</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>318</y>
     <width>30</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>数量</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>198</y>
     <width>30</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>作者</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>438</y>
     <width>45</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ISBN</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="bookCount">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>312</y>
     <width>400</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="ISBNnumber">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>431</y>
     <width>400</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="publishHouse">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>550</y>
     <width>400</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="writerName">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>193</y>
     <width>400</width>
     <height>25</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QLineEdit" name="bookName">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>70</y>
     <width>400</width>
     <height>25</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>71</y>
     <width>30</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>书名</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
